pkg/gsclient/gsclient.go

```diff
@@ -157,28 +152,17 @@ func (c *Client) DeleteCluster(ctx context.Context, clusterID string) error {
 	return nil
 }
 
-func (c *Client) GetKubeconfig(ctx context.Context, clusterID string) (string, error) {
+func (c *Client) CreateKubeconfig(ctx context.Context, clusterID, kubeconfigPath string) error {
 
 	// TODO: extract and structure all these hardcoded values
-	args := fmt.Sprintf("--output=json create kubeconfig --cluster=%s --certificate-organizations system:masters", clusterID)
-	output, err := c.runWithGsctl(args)
-	if err != nil {
-		return "", microerror.Mask(err)
-	}
+	args := fmt.Sprintf("create kubeconfig --cluster=%s --certificate-organizations system:masters --force --self-contained %s", clusterID, kubeconfigPath)
```

Comment:
We gotta clean up this arg handling more, but outside of the scope of this PR :)